### PR TITLE
Covid tracker in Vue.js article

### PR DESCRIPTION
**Pull request by callumacrae**

### CODE REVIEW

File: articles/covid-tracker-in-vue.md [hunk 1]
```diff
@@ -0,0 +1,1100 @@
...
+</svg>
+
+The two attributes added in the last example help us here. `alignment-baseline`
+says where the text should be positioned vertically by specifying which baseline
+of the element should be used. `text-anchor` does a similar thing, but
+horizontally instead of vertically.
+
+Here's a quick demo:
+
+@todo WHY ISN'T ALIGNMENT BASELINE WORKING!
```
Comment (remotelydev) [inline articles/covid-tracker-in-vue.md:228]: Some rendering thing? Have you tried nextTick()?

File: articles/covid-tracker-in-vue.md [hunk 2]
```diff
@@ -0,0 +1,1100 @@
...
+    :transform="`translate(0, ${position * 60})`"
+    :country="country"
+    :max-value="maxValue"
+    :value="value"
+    :chart-width="chartWidth"
+  />
+</transition-group>
+```
+
+You could even just call it `i` if you wanted!
```
Comment (remotelydev) [inline articles/covid-tracker-in-vue.md:867]: want?

File: articles/covid-tracker-in-vue.md [hunk 3]
```diff
@@ -0,0 +1,1100 @@
...
+handling the transition logic for every single bar at the same time in the main
+file.
+
+So how do we transition state?
+
+There's a few different ways we can approach this, but the one we'll go for is
+as follows:
+
+- Add a new property of the data object of the ChartBar component called
+  `tweenedValue`.
```
Comment (remotelydev) [inline articles/covid-tracker-in-vue.md:920]: Maybe explain `tweened` earlier. I was confused

File: articles/covid-tracker-in-vue.md [hunk 4]
```diff
@@ -0,0 +1,1100 @@
...
+    :animation-duration="0.3"
+  />
+</div>
+
+You can now see that when you change the day, the numbers are animated as well
+as the bar widths.
+
+The code on GitHub contains a couple extra features such as configurable
+animation time and smarter bar heights—here's the link again:
+[callumacrae/covid-visualisation](https://github.com/callumacrae/covid-visualisations).
```
Comment (remotelydev) [inline articles/covid-tracker-in-vue.md:1075]: Maybe host it somewhere?

File: articles/covid-tracker-in-vue.md (hunk 1 above)
Comment (callumacrae) [reply articles/covid-tracker-in-vue.md:228]: it looks like a browser bug - it doesn't work at all in firefox 🤔

File: articles/covid-tracker-in-vue.md (hunk 4 above)
Comment (callumacrae) [reply articles/covid-tracker-in-vue.md]: that's the article! 😝

File: articles/covid-tracker-in-vue.md (hunk 1 above)
Comment (callumacrae) [reply articles/covid-tracker-in-vue.md]: fixed by changing to dominant-baseline

**Pull request merged**